src/main/msp/msp.c

```diff
@@ -1808,21 +1808,7 @@ case MSP_NAME:
         break;
 
     case MSP_SENSOR_ALIGNMENT: {
-        uint8_t gyroAlignment;
-#ifdef USE_MULTI_GYRO
-        switch (gyroConfig()->gyro_to_use) {
-        case GYRO_CONFIG_USE_GYRO_2:
-            gyroAlignment = gyroDeviceConfig(1)->alignment;
-            break;
-        case GYRO_CONFIG_USE_GYRO_BOTH:
-            // for dual-gyro in "BOTH" mode we only read/write gyro 0
-        default:
-            gyroAlignment = gyroDeviceConfig(0)->alignment;
-            break;
-        }
-#else
-        gyroAlignment = gyroDeviceConfig(0)->alignment;
-#endif
+        uint8_t gyroAlignment = gyroDeviceConfig(0)->alignment;
```

Comment:
We loose the ability to set alignment for second gyro here.

Reply:
Also add `gyro_enabled_bitmask` here as last parameter in both MSP messages.

Reply:
Well, just the ability to set this in MSP. But adding the bitmask to the MSP would break MSP versioning, I could add it, but it will require an MSP revision. We will also need to think about how we handle alignment going forward, should probably have MSP send which IMU it wants to align along with how it wants to align it.

Reply:
Using GYRO_COUNT and DEFAULT_GYRO_ENABLED handles the config.h files. Now the question is how we want to handle the MSP messages. We will need both the gyro_enabled bitmask, and a way to set a specific IMUs alignment (somehow need to make this so you choose and index and an alignment), however you think is best I'll go forward with.

Reply:
Would we need to add an msp_gyro_to_align value?

Reply:
Needs update in `MSP_[SET_]SENSOR_ALIGNMENT`

Reply:
> Needs update in `MSP_[SET_]SENSOR_ALIGNMENT`

I may need to be pointed in the right direction to get this done.

Reply:
Not sure if sufficient or need to have an array of 8 elements or variable length. Like

![image](https://github.com/user-attachments/assets/e776ca86-e634-49a4-8c23-984299f4944c)

Reply:
You could do that, or if we could somehow pick which IMU we want to change we could just have 1 setting to pick the IMU and another to give settings for that IMU.

Reply:
Think in GUI we like an `overview` of available gyro's with related [custom] alignment settings and a way to set active / nonactive gyro's on the fly.

Reply:
Perhaps reuse gyro_to_use to contain gyro_count or we can derive count using mask.

Reply:
reusing gyro_to_use would break backwords compatibility, we could just have a gyro_count that would just contain the gyro count, just as a message we send (wouldn't make sense to modify this though). From the mask we would only see what the highest gyro count we have enabled is... unless I'm missing something.

Reply:
We like to remember alignment settings when re-enabling disabled gyro